config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  # get 'pages/hello'
+  # root 'pages#hello'
+  root 'users#index'
+  resources :users, only: [:index, :show] do
+  resources :posts, only: [:index, :show]
```

Comment:
- You have done well so far, but please note that you haven't configured your routes like the previous reviewers suggested , the post's routes should be nested under the users routes. Please fix this 😄 
```ruby

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
@mckent05 I have already made the changes and configured my routes as the previous reviewer suggested.

Reply:
Fixed. 
https://github.com/BHObserver/blog-app/blob/b6fb4f2ac46c12ac7e556c8516c58c2598e09b24/config/routes.rb#L4-L7